app/loader.cpp

```diff
@@ -32,6 +32,8 @@
 #include <QDebug>
 
 #include <qgsprojectviewsettings.h>
+#include <QStandardPaths>
+#include <QTimer>
```

Comment:
needed?

Reply:
`QStandardPaths` and `QTimer` are used by this class. I had to add it there otherwise it would not compile. It seems that it found this object before via hierarchy of includes coming from inpututils->positionkit->..., however when I removed positionkit from utils, few places needed new imports.